### PR TITLE
Fixes #35497 - New host UI - Link to host group

### PR DESCRIPTION
**Pull request by stejskalleos**
Screenshot from 2022-09-07 14-26-09

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js [hunk 1]
```diff
@@ -106,7 +108,29 @@ const DetailsCard = ({
...
+                &nbsp;
+                <a
+                  href={foremanUrl(
+                    `/hosts?search=hostgroup="${hostgroupName}"`
+                  )}
+                  className="disabled-text"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <i className="fa fa-search" />
```
Comment (stejskalleos) [inline webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js]: I guess search icon here for is not the best option but I couldn't came up with anything better,
I'm open to suggestions!

Comment (ares) [reply webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js]: how about `pficon-server-group` if we have that available?

**Comment by ares** [comment]
I haven't tested but would like to ask, what does it do when a user does not have a permission to view that host group?

**Review by MariaAga — CHANGES_REQUESTED** [review]
Thanks! can you please use https://www.patternfly.org/v4/components/button in its inline link state?
Also from what I see in the design there should be an edit icon, and then the url on the name itself should be linked to the host search.
https://marvelapp.com/prototype/995cc27/screen/77850645
Please use pfs react icons, here it should be PencilAltIcon
We use  pf components for consistency in looks and qe testing

**Comment by stejskalleos** [comment]
> I haven't tested but would like to ask, what does it do when a user does not have a permission to view that host group?

After user clicks on the edit icon he's redirected to the host group edit page with permission denied error

@MariaAga I updated PR however I failed with the styling of components, https://marvelapp.com/prototype/995cc27/screen/77850645 shows design but I wasn't able to do it with PF4, I couldn't even make the icon in different color, [docs for it](https://www.patternfly.org/v4/guidelines/colors) shows some variables but I have no idea how to implement it. Do we have any source code sample somewhere?

**Comment by MariaAga** [comment]
For the edit button, the `variant` should be `variant="plain"` instead of a link to fix the color.
Also the button should not be shown if there is no host group
Screenshot from 2022-09-14 09-57-29

**Comment by stejskalleos** [comment]
Icon & empty state fixed

**Review by MariaAga — CHANGES_REQUESTED** [review]
Sorry, just noticed now that in the design the pencil is aligned to the right, other than that everything is working as it should

**Comment by stejskalleos** [comment]
[test katello]

**Comment by stejskalleos** [comment]
> Sorry, just noticed now that in the design the pencil is aligned to the right, other than that everything is working as it should

fixed

File: webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js [hunk 2]
```diff
@@ -106,7 +109,39 @@ const DetailsCard = ({
                 emptyState={<DefaultLoaderEmptyState />}
                 status={status}
               >
-                {hostgroupName}
+                {hostgroupId && (
+                  <div className="pf-u-display-flex pf-u-justify-content-space-between">
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js]: There isn't a documentation in the pf4 website but this should also be using pf4 components
https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/layouts/Flex/examples/Flex.md

**Review by MariaAga — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**